modules/tinymce/src/themes/silver/main/ts/api/Events.ts

```diff
@@ -24,11 +25,23 @@ const fireTextColorChange = (editor: Editor, data: { name: string; color: string
   editor.dispatch('TextColorChange', data);
 };
 
+const fireAfterProgressState = (editor: Editor, state: boolean): void => {
```

Comment:
I started to move some until I realised how many there were... so I figured I might as well keep what I had moved.

Reply:
Yeah, events and options usage directly in core/theme will be fun to fix.